index.js

```diff
@@ -309,15 +316,15 @@
 
       return bbox
     }
-    
+
     // Private - replace D3JS 3.X d3.functor() function
     function functor(v) {
-    	return typeof v === "function" ? v : function() {
+      return typeof v === 'function' ? v : function() {
         return v
-    	}
+      }
     }
 
     return tip
-  };
-
+  }
+// eslint-disable-next-line semi
```

Comment:
I left in that last external semicolon because I was uncertain about how people might be bundling this raw source file. I've seen a rare edge case where a build broke when switching build tools. The previous tool was one that manually inserted semis between files while the newer one didnt, and a library without a trailing semi had its neighbour fed into it by accident.

I suppose with that (very) edge case in mind it couldn't hurt to have another one before the library as well.